src/options/components/App.js

```diff
@@ -0,0 +1,107 @@
+import React, { useEffect, useState } from "react";
+import JsonConfig from "./JsonConfig";
+import VisualConfig from "./VisualConfig";
+
+const defaultPrettierOptions = {
+  arrowParens: "avoid",
+  bracketSpacing: true,
+  jsxBracketSameLine: false,
+  printWidth: 80,
+  semi: true,
+  singleQuote: false,
+  tabWidth: 2,
+  trailingComma: "none",
+  useTabs: false
+};
+
+const defaultOptions = {
+  json: {
+    config: "",
+    enable: false
+  },
+  prettier: defaultPrettierOptions
```

Comment:
Sooo after suggesting we scope the options, I realized that `options` is super generic and it gets really confusing to have a nested key called `options`. Just want to make sure this is cool with everyone else - best to fix it now before v1!